include/openssl/asn1.h

```diff
@@ -40,7 +40,7 @@ extern "C" {
 
 # define V_ASN1_CONSTRUCTED              0x20
 # define V_ASN1_PRIMITIVE_TAG            0x1f
-# define V_ASN1_PRIMATIVE_TAG            0x1f
+# define V_ASN1_PRIMITIVE_TAG            0x1f
```

Comment:
Note that the correct one is already above is. This is for compatibility reasons because this is a public header file. Maybe we should add a comment explaining that?